packages/react/src/components/iconCard/IconCard.tsx

```diff
@@ -87,12 +91,19 @@ export const IconCard: React.FunctionComponent<IconCardProps & Omit<React.HTMLPr
                     onClick={handleCardClick}
                     aria-expanded={isOpen}
                 >
-                    <Svg
-                        svgName={svgName}
-                        svgClass={classNames('logo overflow-hidden mod-rounded-border-4 icon mr3', {
-                            'mod-72': !small,
-                            'mod-40': !!small,
-                        })}
+                    <SvgChild
+                        {...({
+                            svgChild,
+                            svgName,
+                            svgClass: classNames(
+                                'logo overflow-hidden mod-rounded-border-4 icon mr3',
+                                {
+                                    'mod-72': !small,
+                                    'mod-40': !!small,
+                                },
+                                svgClass
+                            ),
+                        } as SvgChildProps)}

```

Comment:
Same here, if it works

```suggestion
                    <SvgChild
                        svgChild={svgChild}
                        svgName={svgName}
                        svgClass={classNames(
                            'logo overflow-hidden mod-rounded-border-4 icon mr3',
                            {
                                'mod-72': !small,
                                'mod-40': !!small,
                            },
                            svgClass
                        )}
```